scripts/brp-remove-libtool-files

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# If using normal root, avoid changing anything.
+if [ -z "$RPM_BUILD_ROOT" -o "$RPM_BUILD_ROOT" = "/" ]; then
+	exit 0
+fi
+
+find "$RPM_BUILD_ROOT" \( -type f -o -type l \) -name \*.la  -print0 |
+xargs  --no-run-if-empty -0 file -N -L |
+grep "libtool library file" |
+sed -e 's#: libtool library file.*##g' |
```

Comment:
well, to my defense, shell scripting receives little focus and not exactly the language of choice I spend the most time when doing development. Also I've never really grown comfortable with regexp and I truly *hate* perl,  which I rewrote several of these scripts in shell from.. :p 

But at least it does what it's supposed to do and works well at it, so that's good enough for me as a start, but I'd love to see others applying a more sexy final touch to improve the scripts. :o)